src/Raven.Server/Documents/Indexes/Index.cs

```diff
@@ -1290,7 +1299,9 @@ private void HandleIndexCorruption(IndexingStatsScope stats, Exception e)
             {
                 // it can be a transient error, we are going to unload the database and do not error the index yet
                 // let's stop the indexing thread
-                _indexDisabled = true;  
+                _indexDisabled = true;
```

Comment:
even though the lock isn't needed, it is better to add it, for consistency. 
Otherwise, we'll get analyzer warnings